### PR TITLE
Release notes 2 1 18

### PR DESCRIPTION
**Pull request by jfrontain**
## Purpose of this pull request

This pull request add release notes for the Magento Commerce 2.1.1.8 and Magento Open Source 2.1.18 releases. 

## Affected DevDocs pages

[Magento Commerce 2.1.1.8 Release Notes](https://devdocs.magento.com/guides/v2.1/release-notes/ReleaseNotes2.1.18EE.html)

[Magento Open Source 2.1.1.8 Release Notes](https://devdocs.magento.com/guides/v2.1/release-notes/ReleaseNotes2.1.17CE.html)

whatsnew
Added release notes for the Magento 2.1.18 release.

### CODE REVIEW

**Comment by magento-cicd2** [comment]
[CLA assistant check](https://cla.dev.magento.com/magento/devdocs?pullRequest=4773) <br/>Thank you for your submission, we really appreciate it. Like many open source projects, we ask that you sign our [Contributor License Agreement](https://cla.dev.magento.com/magento/devdocs?pullRequest=4773) before we can accept your contribution.<br/><hr/>**Jeanne Frontain** seems not to be a GitHub user. You need a GitHub account to be able to sign the CLA. If you have already a GitHub account, please [add the email address used for this commit to your account](https://help.github.com/articles/why-are-my-commits-linked-to-the-wrong-user/#commits-are-not-linked-to-any-user).

File: guides/v2.1/release-notes/ReleaseNotes2.1.18CE.md [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+This release include security enhancements that help close cross-site scripting, arbitrary code execution, and sensitive data disclosure vulnerabilities as well as other security issues. No confirmed attacks related to these issues have occurred to date. However, certain vulnerabilities can potentially be exploited to access customer information or take over administrator sessions. We recommend that you upgrade your Magento software to the latest version as soon as possible.
+
+
+
+
+
+
+## Highlights
+
+See [Magento Security Center](https://magento.com/security/patches/magento-2.3.2-2.2.9-and-2.1.18-security-update) for a comprehensive discussion of these issues. All exploitable security issues fixed in this release (2.1.18) have been ported to 2.3.2, 2.2.9, 1.14.4.2, and 1.9.4.2, as appropriate.
```
Comment (meker12) [inline guides/v2.1/release-notes/ReleaseNotes2.1.18CE.md]: Not sure whether "these issues" is referring to the bulleted items that follow, or the security issues you mentioned in the previous paragraph.

File: guides/v2.1/release-notes/ReleaseNotes2.1.18CE.md [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+{:.bs-callout .bs-callout-note}
+The Magento Open Source 2.1.18 software release marks the final supported software release for Magento Open Source version 2.1.x. Magento Open Source 2.1.x will no longer receive security updates or product quality fixes now that its support window has expired.
+
+To maintain the performance and security of your site, we advise you to upgrade to the latest version of Magento Open Source. We encourage you to reach out to your partners, developers, or Magento Customer Success contact for more information on upgrading your site.
+
+
+We are pleased to present {{site.data.var.ce}} 2.1.18. This release includes multiple enhancements to product security. 
+
+
+This release include security enhancements that help close cross-site scripting, arbitrary code execution, and sensitive data disclosure vulnerabilities as well as other security issues. No confirmed attacks related to these issues have occurred to date. However, certain vulnerabilities can potentially be exploited to access customer information or take over administrator sessions. We recommend that you upgrade your Magento software to the latest version as soon as possible.
```
Comment (meker12) [inline guides/v2.1/release-notes/ReleaseNotes2.1.18CE.md]: It seems like the "This release..." paragraph might fit better under the highlights section.  

Also, in the opening paragraph of the release notes, you say: ``` we advise you to upgrade to the latest version of Magento Open Source. ```

This info is repeated in the last sentence: We recommend that you upgrade ....

File: guides/v2.1/release-notes/ReleaseNotes2.1.18CE.md [hunk 3]
```diff
@@ -0,0 +1,65 @@
...
+
+
+
+*Release notes published on June 25, 2019.*
+
+
+{:.bs-callout .bs-callout-note}
+The Magento Open Source 2.1.18 software release marks the final supported software release for Magento Open Source version 2.1.x. Magento Open Source 2.1.x will no longer receive security updates or product quality fixes now that its support window has expired.
+
+To maintain the performance and security of your site, we advise you to upgrade to the latest version of Magento Open Source. We encourage you to reach out to your partners, developers, or Magento Customer Success contact for more information on upgrading your site.
```
[the remark this answers is not in the document]
Comment (jfrontain) [reply guides/v2.1/release-notes/ReleaseNotes2.1.18CE.md]: Hey Margaret, am leaving this on the table -- it's personal style, not grammatical accuracy

File: guides/v2.1/release-notes/ReleaseNotes2.1.18CE.md [hunk 4]
```diff
@@ -0,0 +1,65 @@
...
+
+
+
+## Highlights
+
+See [Magento Security Center](https://magento.com/security/patches/magento-2.3.2-2.2.9-and-2.1.18-security-update) for a comprehensive discussion of these issues. All exploitable security issues fixed in this release (2.1.18) have been ported to 2.3.2, 2.2.9, 1.14.4.2, and 1.9.4.2, as appropriate. 
+
+
+
+<!--- MAGETWO-98950 -->* The CGI URL gateway endpoint in the UPS module has been updated  from HTTP to HTTPS in response to the disablement of the HTTP gateway by UPS in mid-2019. See [Magento User Guide](https://docs.magento.com/m2/ee/user_guide/shipping/ups.html) for a discussion of using the UPS shipment method. Shipping method configuration settings are described in the [Shipping methods](https://docs.magento.com/m2/ee/user_guide/configuration/sales/shipping-methods.html#UPS).
```
[the remark this answers is not in the document]
Comment (jfrontain) [reply guides/v2.1/release-notes/ReleaseNotes2.1.18CE.md]: Style issue again -- will remove the extra space, though

File: guides/v2.1/release-notes/ReleaseNotes2.1.18CE.md (hunk 1 above)
Comment (jfrontain) [reply guides/v2.1/release-notes/ReleaseNotes2.1.18CE.md]: will add "security" before "issue"

File: guides/v2.1/release-notes/ReleaseNotes2.1.18CE.md (hunk 2 above)
Comment (jfrontain) [reply guides/v2.1/release-notes/ReleaseNotes2.1.18CE.md]: Will move section

File: guides/v2.1/release-notes/ReleaseNotes2.1.18CE.md [hunk 5]
```diff
@@ -0,0 +1,65 @@
...
+The Magento Open Source 2.1.18 software release marks the final supported software release for Magento Open Source version 2.1.x. Magento Open Source 2.1.x will no longer receive security updates or product quality fixes now that its support window has expired.
+
+To maintain the performance and security of your site, we advise you to upgrade to the latest version of Magento Open Source. We encourage you to reach out to your partners, developers, or Magento Customer Success contact for more information on upgrading your site.
+
+
+We are pleased to present {{site.data.var.ce}} 2.1.18. This release includes multiple enhancements to product security. 
+
+
+This release include security enhancements that help close cross-site scripting, arbitrary code execution, and sensitive data disclosure vulnerabilities as well as other security issues. No confirmed attacks related to these issues have occurred to date. However, certain vulnerabilities can potentially be exploited to access customer information or take over administrator sessions. We recommend that you upgrade your Magento software to the latest version as soon as possible.
+
```
[the remark this answers is not in the document]
Comment (jfrontain) [reply guides/v2.1/release-notes/ReleaseNotes2.1.18CE.md]: wil remove blank lines

File: guides/v2.1/release-notes/ReleaseNotes2.1.18EE.md [hunk 6]
```diff
@@ -0,0 +1,62 @@
...
+group: release-notes
+title: Magento Commerce 2.1.18 Release Notes
+---
+
+
+*Release notes published on June 25, 2019.*
+
+
+{:.bs-callout .bs-callout-note}
+The Magento Commerce 2.1.18 software release marks the final supported software release for Magento Commerce version 2.1.x. Magento Commerce 2.1.x will no longer receive security updates or product quality fixes now that its support window has expired.
```
Comment (meker12) [inline guides/v2.1/release-notes/ReleaseNotes2.1.18EE.md]:
```suggestion
The Magento Commerce 2.1.18 software release marks the final supported software release for Magento Commerce version 2.1.x. Magento Commerce 2.1.x will no longer receive security updates or product quality fixes now that its support window has expired. Test.
```

**Pull request merged**